internal/command/run.go

```diff
@@ -106,6 +108,8 @@ func newRunCmd() *runCmd {
 		"include a string as an input")
 	cmd.Flags().StringVar(&cmd.lookupInputStr, "lookup-input-str", "",
 		"if a run can not be found, try to find a run with this value as input-string")
+	cmd.Flags().UintVarP(&cmd.buildWorkers, "tasks", "t", 1,
```

Comment:
Please validate if a valid value was passed for the parameter and otherwise fail with an error.
Running `baur run -t 0` would currently hang forever